services/catalog/tests/test_authentication_catalog.py

```diff
@@ -976,6 +976,83 @@ def test_fails_without_good_perms(
             assert response.status_code == HTTP_401_UNAUTHORIZED
 
 
+class TestAuthenticationSearchInCollection:
+
+    search_params = {"filter-lang": "cql2-text", "filter": "width=2500"}
+    test_json = {
```

Comment:
A more realistic test value for search would be to search features by id (name).